config/rbac/rag_job_rbac.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
seems this file is useless? as we will create default clusterrole during helm install